### PR TITLE
Fixing Makefile PROJECT names

### PR DESCRIPTION
**Pull request by amintasvrp**
## This solution refers to which of the apps?

All OWASP Top 10 (2021) apps.

## What changes were made ?

Changed the project name in the configuration files (MAKEFILE) of the OWASP Top 10 apps of 2021. In this sense, it consists of a contribution to solve the topic "Fix typo in deployments files(wrong owasp numeration)" of issue #561, and issue #587.

## Did you test your changes? What commands did you run?

The Make command was executed, before and after the changes, to ensure that the compilation behavior was maintained.

### CODE REVIEW

File: owasp-top10-2021-apps/a2/snake-pro/Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ GODEP ?= $(GOBIN)/dep
 GOLINT ?= $(GOBIN)/golint
 GOSEC ?= $(GOBIN)/gosec
 
-INSECUREBIN ?= snakeProA3
+INSECUREBIN ?= A2 SnakePro
```
Comment (fguisso) [inline owasp-top10-2021-apps/a2/snake-pro/Makefile:12]: Please, change for `snakeProA2`

**Review by fguisso — CHANGES_REQUESTED** [review]

File: owasp-top10-2021-apps/a2/snake-pro/Makefile (hunk 1 above)
Comment (amintasvrp) [reply owasp-top10-2021-apps/a2/snake-pro/Makefile]: Changes done in 576c2bc.

**Review by fguisso — APPROVED** [review]

**Comment by fguisso** [comment]
Awesome @amintasvrp thanks for your contribution.

**Comment by amintasvrp** [comment]
Thank you, @fguisso !
Waiting for Globo Hacktoberfest approve.

image

**Comment by amintasvrp** [comment]
@fguisso, Globo Hacktoberfest website is not recognizing approved PR. Was there a problem?
image

**Review by henriporto — APPROVED** [review]
nice job! 👍

**Pull request merged**